File: 2014-10-CSharp-Fundamentals-1/5. Conditional Statements/NestedIf/NestedIf.cs
using System;

class NestedIf
{
    static void Main()
    {
        Console.WriteLine("Enter two numbers (on separate lines).");

        int first = int.Parse(Console.ReadLine());
        int second = int.Parse(Console.ReadLine());

        if (first == second)
        {
            Console.WriteLine("These two numbers are equal.");
        }
        else
        {
            if (first > second)
            {
                Console.WriteLine("The first number is greater.");
            }
            else
            {
                Console.WriteLine("The second number is greater.");
            }
        }
    }
}

File: 2015-10-Java-Fundamentals-1/05. Conditional-statements/demos/NestedIf.java
import java.text.MessageFormat;
import java.util.Scanner;

public class NestedIf {
    public static void main(String[] args) {
        writeLine("Enter two numbers (on separate lines).");

        Scanner scanner = new Scanner(System.in);
        int first = scanner.nextInt();
        int second = scanner.nextInt();

        if (first == second) {
            writeLine("These two numbers are equal.");
        }
        else {
            if (first > second) {
                writeLine("The first number is greater.");
            }
            else {
                writeLine("The second number is greater.");
            }
        }
    }

    public static void writeLine(Object obj) {
        System.out.println(obj);
    }
}